.tool-versions

```diff
@@ -1,2 +1,3 @@
-nodejs 18.13.0 LTS
-nvm >=0.39.0
\ No newline at end of file
+nodejs 18.14.0
+
+nvm 0.39.0
```

Comment:
Could you remove this line? As you're using nvm to manage the Node version

Reply:
Sorry, Do you mean remove this file ?

Reply:
Fixed in https://github.com/suphanatjarukulgowit/react-nimble-survey/pull/32/commits/d93dd5be70c60066391418af6dc3c7a418e04a01